### ISSUE TITLE
Job with -l nodes=0 is accepted but causes errors

### ISSUE THREAD

**Issue by npf** [oar-team/oar#133]
oar.log:
```
[debug] [2017-03-29 06:25:32.849] [oar_sched_gantt_with_timesharing] Begin phase 2 (waiting jobs)
[debug] [2017-03-29 06:25:32.852] [oar_sched_gantt_with_timesharing] [1222052] start scheduling
[debug] [2017-03-29 06:25:32.989] [oar_sched_gantt_with_timesharing] [1222052] find_first_hole with a timeout of 14.75
[debug] [2017-03-29 06:25:33.009] [oar_sched_gantt_with_timesharing] [1222052] add job occupation in gantt of container 0
DBD::Pg::db do failed: ERROR:  syntax error at or near ")"
LINE 2:               VALUES (1222203,)
                                      ^ at /usr/share/perl5/OAR/IO.pm line 6318.
[debug] [2017-03-29 06:25:33.026] [oar_sched_gantt_with_timesharing] [1222052] end scheduling
```

Job details:
```
oarstat -fj 1222052
Job_Id: 1222052
    job_array_id = 1222052
    job_array_index = 1
    name = 
    project = default
    owner = nrampon
    state = Waiting
    wanted_resources = -l "{type = 'default'}/host=0,walltime=2:0:0" 
    types = 
    dependencies = 
    assigned_resources = 
    assigned_hostnames = 
    queue = default
    command = 
    launchingDirectory = /home/nrampon
    stdout_file = OAR.1222052.stdout
    stderr_file = OAR.1222052.stderr
    jobType = INTERACTIVE
    properties = (maintenance = 'NO') AND production = 'NO'
    reservation = None
    walltime = 2:0:0
    submissionTime = 2017-03-26 20:48:19
    cpuset_name = nrampon_1222052
    initial_request = oarsub -I -l nodes=0,walltime=2:00
    message = FIFO scheduling OK
    scheduledStart = 2017-03-29 09:52:53
    resubmit_job_id = 0
    events = 
```
Queue is oar_sched_gantt_with_timesharing
OAR version 2.5.8+rc3

**Comment by npf** [comment]
Seems like the find first hole function returns OK but with no resources...

**Comment by npf** [comment]
Inpecting the following line:
https://github.com/oar-team/oar/blob/2.5/sources/core/modules/scheduler/oar_sched_gantt_with_timesharing#L436
indeed report not resources in the

**Comment by npf** [comment]
The issue does not seem to exist with the oar_sched_gantt_with_timesharing_and_quota scheduler

**Comment by npf** [comment]
well with the oar_sched_gantt_with_timesharing_and_quotas scheduler, it is actually worse: no warning but all resources are actually assigned to the job...

**Comment by npf** [comment]
Fix seems ok in 2.5.8+rc5. closing.
